benchmarks/src/sort.rs

```diff
@@ -148,8 +148,9 @@ impl RunOpt {
             println!("Executing '{title}' (sorting by: {expr:?})");
             rundata.start_new_case(title);
             for i in 0..self.common.iterations {
-                let config =
-                    SessionConfig::new().with_target_partitions(self.common.partitions);
+                let config = SessionConfig::new().with_target_partitions(
```

Comment:
I think SessionConfig already defaults to the number of CPUs, so we can probably just avoid setting this at all if `self.common.partitions.is_none()`